codebuild/bin/s2n_codebuild.sh

```diff
@@ -105,6 +105,7 @@ run_unit_tests() {
     cmake . -Bbuild \
             -DCMAKE_PREFIX_PATH=$LIBCRYPTO_ROOT \
             -D${CMAKE_PQ_OPTION} \
+            -DS2N_AWSLC_KYBER_UNSTABLE=ON \
```

Comment:
Nice catch!

Reply:
thanks. now that we have the KEM API, i think it might be time to remove this feature flag altogether and prefer AWS-LC Kyber whenever s2n is built against AWS-LC.

@lrstewart @dkostic  -- what do you think?

Reply:
I've removed the S2N_AWSLC_KYBER_UNSTABLE flag. please let me know if you'd like me to put it back in.

Reply:
Just to be clear, ALL versions of aws-lc support kyber? There's no situation where s2n can be build against aws-lc and the necessary functions don't exist?

Reply:
my understanding was that we'll enable Kyber from AWS-LC by default only when we get a sign-off on our (AWS-LC) security review. So I think we should wait for that (matter of days now).

Reply:
@dkostic -- got it. i'll hold off on merging until I get confirmation from you on sign-off.

Reply:
the aforementioned sign-off has been obtained and relevant code changes have been merged.